### ISSUE TITLE
[Issue]: nvm upgrade does nothing

### ISSUE THREAD

**Issue by mjomble** [coreybutler/nvm-windows#1199]
### What happened?

`nvm debug` tells me that a new version is available:

> An upgrade is available: v1.2.1
> run "nvm upgrade" to update.

If I execute `nvm upgrade`, it just stalls indefinitely and doesn't output anything.
If I eventually press `Ctrl-C`, it says "Installation canceled by user".

### What did you expect to happen?

Something should appear in the output.
Either information about a successful upgrade or a message saying I'm already on the latest version.

Looking at the GitHub releases page, it seems 1.2.1 isn't actually released yet.

### Version

1.2.0 or newer (Default)

### Which version of Windows?

Windows 11+ (Default)

### Which locale?

English (Default)

### Which shell are you running NVM4W in?

PowerShell

### User Permissions?

Administrative Privileges, Elevated (tried both elevated and non-elevated, same result)

### Is Developer Mode enabled?

No (Default)

### Debug Output

```Shell
mjomb is not using admin or elevated rights.

Windows Version:        10.0 (Build 26100)

Windows Developer Mode: UNKNOWN

NVM4W Version:          1.2.0
NVM4W Path:             C:\Users\mjomb\AppData\Roaming\nvm\nvm.exe
NVM4W Settings:         C:\Users\mjomb\AppData\Roaming\nvm\settings.txt
NVM_HOME:               C:\Users\mjomb\AppData\Roaming\nvm
NVM_SYMLINK:            C:\dev\apps\nvm4w\nodejs
Node Installations:     C:\Users\mjomb\AppData\Roaming\nvm
Default Architecture:   64-bit
Mirrors:                No mirrors configured
HTTP Proxy:             none

Total Node.js Versions: 2
Active Node.js Version: v20.11.0

No problems detected.

⚠️  We are currently building Author/Runtime to succeed NVM for Windows.

⚠️  An upgrade is available: v1.2.1
   run "nvm upgrade" to update.


Find help at https://github.com/coreybutler/nvm-windows/wiki/Common-Issues
```

**Comment by coreybutler** [comment]
Some hard coded paths made it into the release... not sure how. I'm working on the fix right now and pulled 1.2.1 down until I can get it fixed.

**Comment by coreybutler** [comment]
I've fixed this and will release it in v1.2.1 as soon as I can tomorrow.
